plugin/pom.xml

```diff
@@ -3,39 +3,34 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.googlecode.gwtquery.plugins</groupId>
-        <artifactId>draggable</artifactId>
+        <groupId>com.arcbees.gquery</groupId>
+        <artifactId>draggable-plugin</artifactId>
         <version>1.1.0-SNAPSHOT</version>
     </parent>
 
     <name>Draggable plugin for gQuery</name>
-    <artifactId>draggable-plugin</artifactId>
+    <artifactId>draggable</artifactId>
```

Comment:
Yeah, try to follow same artifact ID structure that we already have. If the main pom is a problem, use "draggable" in the main pom or "draggable-parent"